docs/content/storage-migrate.md

```diff
@@ -2,81 +2,180 @@
+
+In v0, Porter supported storing data with the filesystem plugin, storing data in flat files in your PORTER_HOME directory, which is now deprecated.
+The Azure Blob and Azure Table storage plugins have been deprecated as well, and instead you should use Azure CosmosDB with the mongodb plugin.
+
+If you would like to use the mongodb-docker plugin (and are not migrating production data), then no further changes are required and you may move to the next step.
+Otherwise, open your Porter config file located in PORTER_HOME, for example ~/.porter/config.yaml, and configure the storage plugin.
+Below is an example of how to configure Porter to use an external Mongo database.
+The example uses an external secret store so that the database connection string is not stored in plain text in the configuration file.
+You may also use an environment variable, such as `${env.MYDB_CONNSTR}`, to provide Porter the connection string.
+
+NOTE: Porter uses the [standard mongodb connection string format](https://www.mongodb.com/docs/drivers/go/current/fundamentals/connection/), which looks like this for a local development server: `mongodb://localhost:27017`.
+
+```yaml
+# ~/.porter/config.yaml
+default-storage: mydb
+
+storage:
+  - name: mydb
+    plugin: mongodb
+    config:
+      # Do not store sensitive data in the Porter config file
+      # Use a secret or environment variable instead
+      url: ${secret.mydb-connstr}
+
+secrets:
+  - name: mysecrets
+    plugin: azure.keyvault
+    config:
+      # Define a secret named "mydb-connstr" in this vault with the mongodb connection string
+      vault: myvault
+```
+
+## 4. Validate Your Porter Configuration
 
-Backup Porter's data before performing the migration. You should always backup
-the Porter home directory, usually **~/.porter**.
+Run `porter version` to verify that you are using v1 of Porter, and then run `porter list` and validate that your configuration file is correct.
+The command should return zero results because you are starting with a new PORTER_HOME and database.
 
-You may have data stored in an additional remote location depending on the
-plugin that you are using. Open up your Porter config file located at
-~/.porter/config.toml.
+```console
+$ porter version
+porter v1.0.0-beta.1
 
-* If you don't have one, congratulations!, you are using the mongodb-docker plugin 
-  and all of your data is in Porter home.
-* If you do have a config file, take a look and determine if you are using a 
-  different storage plugin.
-  
-NOTE: Only storage plugins have data that requires migration, not secret plugins.
+$ porter list
+-------------------------------------------------------------------------------------
+  NAMESPACE  NAME                  CREATED     MODIFIED    LAST ACTION  LAST STATUS
+-------------------------------------------------------------------------------------
+```
 
-### Azure Plugin
+## 5. Migrate Your Data
```

Comment:
Do you think we need to call out that if their old data contains any sensitive data, they also need to configure a secret store?